cpp/src/strings/convert/convert_durations.cu

```diff
@@ -271,7 +271,7 @@ struct duration_to_string_fn : public duration_to_string_size_fn<T> {
     return str;
   }
 
-  __device__ char* int_to_2digitstr(char* str, int min_digits, int8_t value)
+  __device__ char* int_to_2digitstr(char* str, int, int8_t value)
```

Comment:
Looks like we could change the calling functions to not pass this parameter and remove the argument altogether.

Reply:
Good point, I dropped the parameter entirely.